contrib/endpoints/src/api_manager/service_control/aggregated_test.cc

```diff
@@ -312,20 +312,6 @@ TEST_F(QuotaAllocationTestWithRealClient, AllocateQuotaTest) {
                  [](Status status) { ASSERT_TRUE(status.ok()); });
 }
 
-TEST_F(QuotaAllocationTestWithRealClient, AllocateQuotaFailedTest) {
```

Comment:
It is better if we keep this test case.
First call is passed. I think second call should fail.